CRT.c

```diff
@@ -885,6 +885,7 @@ void CRT_init(const Settings* settings, bool allowUnicode) {
    sigaction (SIGSYS, &act, &old_sig_handler[SIGSYS]);
    sigaction (SIGABRT, &act, &old_sig_handler[SIGABRT]);
 
+   signal(SIGINT, CRT_handleSIGTERM);
```

Comment:
Why this additional handler?

Reply:
ncurses installs its own signal handler for SIGINT, more traditional curses implementations don't.